Boolder/UI/Map/MapboxViewController.swift

```diff
@@ -737,7 +737,7 @@ class MapboxViewController: UIViewController {
                 let cameraOptions = CameraOptions(
                     center: location.coordinate,
                     padding: safePadding,
-                    zoom: 17
+                    zoom: mapView.cameraState.zoom > 17 ? mapView.cameraState.zoom : 17
```

Comment:
It works 👍 

However you can make it even DRYer by using the `max` method, similar to what we did on Android: https://github.com/boolder-org/boolder-android/pull/34/files#diff-9e221e85c01b70d0a1f59bd0e0216873553dcbb9243e29977f57a764fbc6487eR159

Can you make the modification?
Big thanks!